templates/mail/issue/default.tmpl

```diff
@@ -44,11 +44,11 @@
 		{{else if eq .ActionName "merge"}}
 			Merged #{{.Issue.Index}} into {{.Issue.PullRequest.BaseBranch}}.
 		{{else if eq .ActionName "approve"}}
-			<b>@{{.Doer.Name}}</b> approved this pull request.
+			<b>@{{.Doer.Name}}{{if .Doer.FullName}} ({{.Doer.FullName}}){{end}}</b> approved this pull request.
```

Comment:
I don't think it should show full name if still using @username -- it doesn't look right to say
```
@mrsdizzie my full name approved this
```
DEFAULT_SHOW_FULL_NAME should only be for cases where you can actually replace the username and not be used next to the username. See messages from Github as well they will use full name as sender name in the From: field but @username in message body

Reply:
it says `@mrsdizzie (my full name)` approved this, there are parentheses there

Reply:
I still feel similar about it (personally) as the full name would be the sender of the email already -- others could have a different opinion

Reply:
I understand your point, as the name is already in the sender, maybe it's better to just leave the name away?
We could put "Approved this" in it? This already happens when creating issues or merging PR's.

Reply:
I think for the message body when needing the username it is best to leave  out the full name (but it is fine everywhere else where you can replace the username with it).

I'm not sure what you mean by "put approved this in it" since it already has that text in the current template?

Reply:
I mean instead of setting the message `@mrsdizzie my full name approved this` set the message `Approved this`.
There's no need to put the `@mrsdizzie` in that mail, like you say it's already in the sender.

The merge and new issue messages also don't have a `@mrsdizzie merged #issuenr into x` or `@mrsdizzie created #issuenr`. They just say `Merged #issuenr into x` and `Created #issuenr`

Reply:
It wouldn't have the username at all if you use the full name option -- I think it is best to leave the username in the email as is for these where they are ultimately messages in a thread and not an opening or closing message. I think the other templates could have the username too but thats not really related to this change.

I understand your case but for most people it is the username that is recognizable and not the full name -- so removing the username from default templates doesn't feel like an option here.

Reply:
These are also customizable too so its certainly possible to remove the username from a particular instance if you wish I just don't think it would be a good default setting